e2e_tests/provider_cfg/pkcs11/config.toml

```diff
@@ -22,5 +22,6 @@ provider_type = "Pkcs11"
 key_info_manager = "on-disk-manager"
 library_path = "/usr/local/lib/softhsm/libsofthsm2.so"
 user_pin = "123456"
+software_public_operations = true
```

Comment:
Do you think we should keep it that way forever? It would be nice to check that public key operations executed on hardware are working as well if that is the default.
I think it's fine to have the `software_public_operations` only checked manually for now.

Reply:
Yeah, sorry, I'll change that back and add some `config` tests that check the software implementation

Reply:
I've added a couple of tests for the software verification. Do you think we should run all tests against both? As in, do something in `ci.sh` instead of just config tests?